l10n_br_fiscal/demo/company_demo.xml

```diff
@@ -3,10 +3,26 @@
 
     <!-- Sua Empresa -->
     <record id="base.main_company" model="res.company">
-        <field name="cnpj_cpf">97.231.608/0001-69</field>
```

Comment:
Porque deixar em branco esses dados da empresa principal?

Reply:
Pois a ideia é deixar a empresa principal só com os dados de demo do core.

Reply:
Poderia deixar os dados por enquanto, depois em outro PR quando for criado a empresa do lucro real, ai sim remover os dados do main_company, porque muitos testes da localização estão sendo feitos com o main_company e precisa destes dados, e teria que corrigi-los